_data/navigation.yml

```diff
@@ -85,7 +85,10 @@ docs_integrate:
       #  url: /integrate/event-introduction
       - title: "Use Cases"
         url: /integrate/use-cases
-
+  - title: Get in touch with other Identities
+    children:
+      - title: "Process received enmeshed onboarding package and create relationship"
```

Comment:
I encounter the wording "onboarding package" the first time now. Is this the RelationshipTemplate's content?